Makefile

```diff
@@ -179,8 +179,11 @@ cluster.clean: ## Remove unused Docker volumes and networks
 	docker network prune --force
 	docker system prune --volumes --force
 
+linters:
+	./bin/golangci-lint run ./... --timeout=5m
 
-
+linters.install:
+	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s
```

Comment:
This is a moving target, could we install a released version please?

Reply:
As I know we even can't specify a version in the config. I took it from the doc https://golangci-lint.run/usage/install/#local-installation

Reply:
I would use a tag, e.g. https://github.com/golangci/golangci-lint/tree/v1.45.2 but it's NBD.

Reply:
IMO, we don't need install command as part of make file. We can keep only mostly used command as part of Make file and add instruction or link to how to install golangci-lint at README page. WDYT?

Reply:
> we don't need install command as part of make file

I think you are right

> add instruction or link on how to install golangci-lint at README page

golangci-lint is optional and I think we can remove it and that's it. Let's start from that.